superflore/generators/ebuild/run.py

```diff
@@ -31,6 +31,7 @@
 def link_existing_files():
     global overlay
     global mode
+    global preserve_existing
```

Comment:
I don't see this instance being used. And these globals are ugly. They should get cleaned up when swapping out argparse.